### PR TITLE
add measurement for kpa and sks

### PR DESCRIPTION
**Pull request by zyjiaobj**
# Changes

- add kpa measurement
- add sks measurement
- add ready/fail/notReady rate
- fix ready/fail/notReady count bug
- change go-bindata library to a newer one
- update document

### CODE REVIEW

File: pkg/command/service/measure.go [hunk 1]
```diff
@@ -147,20 +152,37 @@ kperf service measure --svc-perfix svc --range 1,200 --namespace ns --concurrenc
...
+							continue
 						}
 						svc := j[0]
 						svcNs := j[1]
 						svcIns, err := servingClient.Services(svcNs).Get(context.TODO(), svc, metav1.GetOptions{})
 						if err != nil {
 							fmt.Printf("failed to get Knative Service %s\n", err)
+							if strings.Contains(err.Error(), "not found") {
+								currentMeasureResult.notFoundCount++
+								fmt.Println(currentMeasureResult.notFoundCount)
```
Comment (gyohuangxin) [inline pkg/command/service/measure.go:167]: Just print the `notFoundCount`? Or is it a debug print?

**Review by maximilien — APPROVED** [review]
/ok-to-test
/lgtm
/hold to allow others to review

File: pkg/command/service/measure.go (hunk 1 above)
Comment (zyjiaobj) [reply pkg/command/service/measure.go]: Thanks! fixed

**Review by gyohuangxin — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ $ kperf service measure --namespace ktest-1 --svc-prefix ktest --range 0,9  --ve
 [Verbose] Service ktest-0: Overall Service Ready Duration is 54s/54.000000s
 ......
 -------- Measurement --------
-Total: 10 | Ready: 10 Fail: 0 NotFound: 0 
+Total: 10 | Ready: 10 Fail: 0 NotFound: 0 Fail: 0
```
Comment (gyohuangxin) [inline README.md:102]: Is it a duplicate `Fail: 0`?

Comment (zyjiaobj) [reply README.md]: Thanks & fixed

**Pull request merged**